packages/web/src/common/store/pages/signon/sagas.js

```diff
@@ -359,20 +359,20 @@ function* signUp() {
 
   const signOn = yield select(getSignOn)
   const location = yield call(getCityAndRegion)
-  const createUserMetadata = {
-    name: signOn.name.value.trim(),
-    handle: signOn.handle.value,
-    profilePicture: (signOn.profileImage && signOn.profileImage.file) || null,
-    coverPhoto: (signOn.coverPhoto && signOn.coverPhoto.file) || null,
-    isVerified: signOn.verified,
-    location
-  }
   const name = signOn.name.value.trim()
   const email = signOn.email.value
   const password = signOn.password.value
   const handle = signOn.handle.value
   const alreadyExisted = signOn.accountAlreadyExisted
   const referrer = signOn.referrer
+  const createUserMetadata = {
+    name,
+    handle,
+    profilePicture: signOn.profileImage?.file || null,
```

Comment:
@DejayJD just want to triple check with you on this line considering some of the hotfixes you did last week touched this stuff

Reply:
Yea this looks fine, just looks like an added ternary? But yeah as long as its still using `.file` its fine 👍